crates/js-component-bindgen/src/transpile_bindgen.rs

```diff
@@ -599,6 +600,20 @@ impl<'a> Instantiator<'a, '_> {
         }
     }
 
+    fn ensure_instance_flags(&mut self) {
```

Comment:
We can just call this `instance_flags` I think, since the `ensure` naming is usually for cases where we need to check it against an already-written list.